src/trivyHelper.ts

```diff
@@ -129,7 +128,7 @@ export function getSummary(trivyStatus: number): string {
             summary = `Found ${total} vulnerabilities -${summaryDetails}`;
             break;
         default:
-            summary = 'An error occurred while scanning the container image for vulnerabilities';
+            summary = 'An error occurred while scanning container image: ' + inputHelper.imageName + ' for vulnerabilities. Please check your image name and try again.';
```

Comment:
That only shows up, I believe, when the run is successful/the task actually runs. If the scan fails right off the bat because it can't find the image (image syntax is incorrect), then you do not get that summary. Then, from a user perspective, the logs don't tell them which image was scanned, which makes it hard to realize your image name was incorrect. 

Further, the current error message doesn't provide a link to the scan result if the scan fails immediately.
![image](https://user-images.githubusercontent.com/7676214/110867633-eb960d80-827b-11eb-8976-00e7ec2a7660.png)
With that said, I propose we do not revert this change, as its logging information that is helpful.

Reply:
@jldeen My bad with the ambiguous sentence.
What I meant was that - This code is for generating the check run summary (which already contains the image name).
The change to log the image name before scanning the image should inform the user which image is being scanned.

So, this change is not required.